### PR TITLE
html and css

### PR DESCRIPTION
**Pull request by yardenporat**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (PeterShershov) [inline index.html:6]: Change to a relevant title ("Dropdown" is fine)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
```
Comment (PeterShershov) [inline index.html:8]: not used

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  </head>
+  <body>
+    <main>
+      <h2>Selection is: <span class="selection"></span></div>
+      <div class="dropdown">
```
Comment (PeterShershov) [inline index.html:13]: should be outside the dropdown opener

File: index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <main>
+      <h2>Selection is: <span class="selection"></span></div>
```
Comment (PeterShershov) [inline index.html:12]: closing tag should be `h2`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      <div class="dropdown">
+        <div class="label">Choose an option:</div>
```
Comment (PeterShershov) [inline index.html:14]: h3?

File: style.css [hunk 6]
```diff
@@ -0,0 +1,32 @@
+.html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 62.5%;
```
Comment (PeterShershov) [inline style.css:5]: explain this decision in comment above.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+body {
+  margin-top: 3rem;
```
Comment (PeterShershov) [inline style.css:9]: put this on the main container

File: style.css [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+  place-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.dropdown {
+  padding: 0.45rem;
+  margin: 0.25rem;
+  border: 1px solid grey;
+  border-radius: 0.5rem;
+  cursor: pointer;
```
Comment (PeterShershov) [inline style.css:20]: move to the item itself and handle item's hover state (darker background)

File: style.css [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  box-shadow: 2px 2px 6px;
+}
+.dropdown-options {
+  display: block;
```
Comment (PeterShershov) [inline style.css:24]: redudant currently

File: style.css [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+}
+.option {
+  padding: 0.25rem;
+  margin: 0.25rem;
+  border-radius: 0.5rem;
+  background-color: white;
+  list-style-type: none;
```
Comment (PeterShershov) [inline style.css:31]: move to `.dropdown-options`

File: style.css (hunk 6 above)
Comment (yardenporat) [reply style.css:5]: 10px final decision

File: index.html [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Animals Dropdown</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <h2>Selection is: <span class="selection"></span></h2>
+      <h3 class="dropdown">
+        <div class="label">Choose your favorite animal:</div>
```
Comment (PeterShershov) [inline index.html:13]: this should be the h3

File: index.html [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+        <ul class="dropdown-options">
```
Comment (PeterShershov) [inline index.html:14]: make it overlayed.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+          <li class="option" id="1">Baboon</li>
+          <li class="option" id="2">Rhino</li>
+          <li class="option" id="3">Cat</li>
```
Comment (PeterShershov) [inline index.html:17]: remove ids

File: style.css [hunk 14]
```diff
@@ -0,0 +1,62 @@
+.html {
```
Comment (PeterShershov) [inline style.css:1]: just `html`

**Review by PeterShershov — APPROVED** [review]
just fix the class name :)

**Pull request merged**